unbxdSearch.js

```diff
@@ -1277,16 +1372,36 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
 
             return this;
         },
-        setDefaultParams: function(params) {
+        setDefaultParams: function (params) {
             var newparams, diff;
             var oldparams = JSON.stringify(params);
             this.options.setDefaultFilters.call(this);
+
+            this.params['extra']['version'] = 'V2';
+
+            if (this.options.facetMultilevel) {
+                this.params['extra']['facet.multilevel'] = 'categoryPath';
```

Comment:
Are these not configurable.

Reply:
ToDo:
In future, there might be more fields for multilevel, hence needs to be changed later